master/internal/api_project.go

```diff
@@ -119,6 +125,12 @@ var defaultRunsTableColumns = []*projectv1.ProjectColumn{
 		Location:    projectv1.LocationType_LOCATION_TYPE_RUN,
 		Type:        projectv1.ColumnType_COLUMN_TYPE_TEXT,
 	},
+	{
+		Column:      "projectId",
+		DisplayName: "Project ID",
+		Location:    projectv1.LocationType_LOCATION_TYPE_RUN,
+		Type:        projectv1.ColumnType_COLUMN_TYPE_TEXT,
```

Comment:
question (non-blocking): is there a reason this is done as text and not a number?

Reply:
Nope. this is supposed to be a number. Thanks!